benchmark/single-source/SetTests.swift

```diff
@@ -98,6 +98,47 @@ public let SetTests = [
     tags: [.validation, .api, .Set],
     setUpFunction: { blackHole([setP, setQ]) }),
 
+  BenchmarkInfo(
+    name: "Set.isStrictSubset.Empty.Int",
+    runFunction: { n in run_SetIsStrictSubsetInt(setE, setAB, true, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setE, setAB]) }),
+  BenchmarkInfo(
+    name: "Set.isStrictSubset.Int.Empty",
+    runFunction: { n in run_SetIsStrictSubsetInt(setAB, setE, false, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setAB, setE]) }),
+  BenchmarkInfo(
+    name: "SetIsStrictSubsetInt0",
+    runFunction: { n in run_SetIsStrictSubsetInt(setAB, setCD, false, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setAB, setCD]) }),
+  BenchmarkInfo(
+    // name: "SetIsStrictSubsetBox0",
+    // runFunction: { n in run_SetIsStrictSubsetInt(setOAB, setOCD, false, 5000 * n) },
+    // tags: [.validation, .api, .Set],
+    // setUpFunction: { blackHole([setOAB, setOCD]) }),
+  BenchmarkInfo(
+    name: "SetIsStrictSubsetInt25",
+    runFunction: { n in run_SetIsStrictSubsetInt(setB, setAB, true, 50 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setB, setAB]) }),
+  // BenchmarkInfo(
+  //   name: "SetIsStrictSubsetBox25",
+  //   runFunction: { n in run_SetIsStrictSubsetInt(setOB, setOAB, true, 50 * n) },
```

Comment:
The problem here is that you'll need a separate `run_SetIsStrictSubsetBox` function for the boxed case -- `setOAB` is of type `Set<Box<Int>>`, not `Set<Int>`.

For example, for the `isSubset` case, we have `run_SetIsSubsetInt` vs `run_SetIsSubsetBox` below.

Reply:
Thank you for reviewing.
I fixed the box variants.